### ISSUE TITLE
Sample Azure template_url's are not working

### ISSUE THREAD

**Issue by hantuzun** [dcos/dcos-launch#54]
The sample azure configs' [\[1\]](https://github.com/dcos/dcos-launch/blob/master/dcos_launch/sample_configs/azure.yaml)[\[2\]](https://github.com/dcos/dcos-launch/blob/master/dcos_launch/sample_configs/azure-with-helper.yaml) template_url is the following value: https://downloads.dcos.io/dcos/testing/master/azure/acs-deploy-1.json

However, DC/OS is no longer serving that URL to public and I was unable to find a Azure Template file working with dcos-launch. The examples' `template_url`'s should be updated.

---

Then I tried the most recent azure template: https://dcosio.azureedge.net/dcos/stable/1.10.0/commit/e38ab2aa282077c8eb7bf103c6fff7b0f08db1a4/azure/acs-1master.azuredeploy.json

Using that template, with either `python cli.py create` on macOS 10.12 or with [the published binary](https://downloads.dcos.io/dcos-launch/bin/linux/dcos-launch) like `./dcos-launch create` on Ubuntu 16.04, I keep getting the following error on template validation: 

```sh
[2017-11-02 13:33:35,671|dcos_launch.platforms.arm|INFO]: Checking deployment parameters vs template before starting...
[2017-11-02 13:33:37,043|dcos_launch.platforms.arm|INFO]: Starting resource group_creation
[2017-11-02 13:33:38,636|dcos_launch.platforms.arm|INFO]: Resource group created: Azure-Resource-Manager-Trial-Deploy
[2017-11-02 13:33:38,637|dcos_launch.platforms.arm|INFO]: Checking with Azure to validate template deployment
Traceback (most recent call last):
  File "cli.py", line 133, in <module>
    sys.exit(main())
  File "cli.py", line 125, in main
    return do_main(args)
  File "cli.py", line 80, in do_main
    write_json(info_path, dcos_launch.get_launcher(config).create())
  File "/dcos-launch/dcos_launch/arm.py", line 38, in create
    self.config.get('tags'))
  File "/dcos-launch/dcos_launch/platforms/arm.py", line 109, in deploy_template_to_new_resource_group
    raise Exception("Template verification failed!\n{}".format(get_all_details(result.error)))
Exception: Template verification failed!
InvalidTemplate: Deployment template validation failed: 'The provided value 'False' for the template parameter 'oauthEnabled' at line '1' and column '21364' is not valid. The parameter value is not part of the allowed value(s): 'true,false'.'.
```

I'm unable to see where this issue is coming from.

I've also tried generating new templates from the master branch of https://github.com/dcos/dcos: https://gist.github.com/hantuzun/2c041732bd888fa3a6d048bbd5b1f375 and tried finding old DC/OS templates in the DC/OS's git history and the Internet with no help.

**Comment by cprovencher** [comment]
The validation error you were getting is because it turns out the parameter oauthEnabled was expecting a boolean as a string instead of a boolean directly as seen in this fix https://github.com/dcos/dcos-launch/pull/189/files
